javascript-carousel/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Carousel</title>
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="view-container">
+
+
+    <div class="row">
+      <div class="col outer-width vertical-center"><i class="fas fa-chevron-left icon-size"></i></div>
+      <div class="col center-width">
+        <!-- div1 -->
+        <div class="view" data-view="1">
+          <img src="images/025.png" alt="">
+          <div class="row">
+            <div class="col">
+              <i class="fas fa-circle"></i>
+              <i class="far fa-circle"></i>
+              <i class="far fa-circle"></i>
+              <i class="far fa-circle"></i>
+              <i class="far fa-circle"></i>
+            </div>
+          </div>
```

Comment:
You should try to do this in a way where there is only one set of dots in the HTML, the only thing there should be multiples of in the HTML is the images themselves.

Reply:
You can probably shift this relatively easily by moving the `data-view` on the image element rather than the container for all of this.